launcher/client/src/components/JobRequest.tsx

```diff
@@ -170,14 +166,16 @@ export const JobRequest = ({
     setIsLoading(false);
   };
 
-  // material ui basic table
-  const [value, setValue] = useState(activeTab);
+  const {
+    state: { activeTab },
+    dispatch,
+  } = useContext(AppContext);
 
-  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
-    setValue(newValue);
+  const handleChange = (event: React.SyntheticEvent, newActiveTab: number) => {
```

Comment:
how is the value of newActiveTab decided or where will that value come from?

Reply:
Check line 199 of the same file.